_includes/seo.html

```diff
@@ -0,0 +1,125 @@
+{% comment %} Store game name for later, if relevant {% endcomment %}
+{%- if true -%}
```

Comment:
why are there  `if true`s here?

Reply:
only way I could find to allow section folding, lol